ApiDoctor.Validation/DocFile.cs

```diff
@@ -1299,7 +1299,7 @@ public bool ValidateNoBrokenLinks(bool includeWarnings, IssueLogger issues, out
                         issues.Error(ValidationErrorCode.LinkDestinationNotFound, $"BookmarkMissing: '[{link.Definition.url}]({link.Text})'. {suggestion}");
                         break;
                     case LinkValidationResult.ParentAboveDocSetPath:
-                        issues.Error(ValidationErrorCode.LinkDestinationOutsideDocSet, $"Relative link outside of doc set: '[{link.Definition.url}]({link.Text})'.");
+                        //Removed this error because of the beta-disclaimer.md file is in the includes [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
```

Comment:
there should be a way to register this as an exception somehow. or make sure the disclaimer file gets checked into the repo so it can be found. i don't think you should remove broken link validation

Reply:
![disclaimer](https://user-images.githubusercontent.com/1641829/52814097-5f20bd00-30ac-11e9-9a3e-c9b4b0c81e8f.PNG)
The disclaimer file is placed in the includes folder. Its in the git repo.

I could make a special case inside the case, to handle this singular case of the disclaimer file.

Reply:
if the file is there, apidoctor shouldn't complain. do you know why it can't find the file?

Reply:
i tried but i'm unable to reproduce this. apidoctor doesn't complain about that link for me. can you confirm your repro and share your doc branch with me?

Reply:
Oddly enough, I can no longer reproduce it too. My suspicion is that the docfx features I added fixed this issue in a cascade. Reverting to earlier state. Thanks for the heads up.